ooquery/ooquery.py

```diff
@@ -63,6 +63,13 @@ def fields(self):
                 fields.append(table_field)
             else:
                 table_field = self.parser.get_table_field(self.table, field)
+                if table_field is None:
+                    raise Exception(
```

Comment:
The error is still a non existing attribute in the table.
I think we should keep the Exception type as "AttributeError"

```suggestion
                    raise AttributeError(
```